dev/tools/gen_defaults/lib/action_chip_template.dart

```diff
@@ -60,10 +60,10 @@ class _${blockName}DefaultsM3 extends ChipThemeData {
   Color? get surfaceTintColor => ${colorOrTransparent("$tokenGroup.container.surface-tint-layer.color")};
 
   @override
-  Color? get checkmarkColor => ${color("$tokenGroup.with-icon.selected.icon.color")};
```

Comment:
these tokens have just been renamed, `X.with-icon.icon.color` and should not be replaced with `null`